src/psyclone/tests/test_files/dynamo0p3/kernels/dead_end/no_really/testkern_qr.F90

```diff
@@ -0,0 +1,87 @@
+! -----------------------------------------------------------------------------
+! BSD 3-Clause License
+!
+! Copyright (c) 2017-2021, Science and Technology Facilities Council
```

Comment:
Is this file moved/copied or is it new? In any case, "2021" should be "2022".

Reply:
It is copied and is just there to make sure there is not a false match of a kernel when there is a hierarchy of directories. I've pretty much replicated what we did in the `dynamo0.1` tests. Updated to 2022, thanks.